app/services/layer-groups.js

```diff
@@ -2,6 +2,16 @@ import Service from '@ember/service';
 import { A } from '@ember/array';
 import { copy } from 'ember-copy';
 
+const areIncomingLayerGroupsDefault = function(layerGroupsToShow, layerGroupsByDefault) {
+  const layersToShowAreDefault = layerGroupsToShow.every(
+    (layerGroupToShow, idx) => layerGroupToShow === layerGroupsByDefault[idx],
+  );
+
+  const currentDiffersFromDefault = layerGroupsToShow.length === layerGroupsByDefault.length;
+
+  return layersToShowAreDefault && currentDiffersFromDefault;
```

Comment:
The naming here is great/intuitive, but I would love some comments explaining why this change was needed.

A little confused on why `currentDiffersFromDefault` would equal visible layer-groups having the same length as the default ones. Wouldn't that be the opposite?